src/sentry/hybridcloud/apigateway/proxy.py

```diff
@@ -157,6 +157,13 @@ def proxy_region_request(
 
     timeout = ENDPOINT_TIMEOUT_OVERRIDE.get(url_name, settings.GATEWAY_PROXY_TIMEOUT)
     metric_tags = {"region": region.name, "url_name": url_name}
+
+    # XXX: See sentry.testutils.pytest.sentry for more information
+    if getattr(settings, "APIGATEWAY_PROXY_SKIP_RELAY") and request.path.startswith(
```

Comment:
Should this not be `getattr(settings, "APIGATEWAY_PROXY_SKIP_RELAY", False)`? Or it needs to be defaulted somewhere. But this will fail with an attribute error in prod from what I can tell.